README.md

```diff
@@ -61,6 +61,14 @@ Derived from job postings or other knowledge, this is where the team works. Defi
 
 * Clang, LLVM, LLDB
 
+## [CompilerWorks](http://http://www.compilerworks.com/about.html#jobs)
```

Comment:
Link has a typo, `http://` is repeated twice.

Reply:
corrected